operator/hack/update-generated-crd.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+function install_crd {
+  if [[ -z "${SKIP_COPY+1}" ]]; then
```

Comment:
Do we use `SKIP_COPY` anywhere? What does it do?

Reply:
It can be used when calling the `manifests` make target. If some value is assigned to `SKIP_COPY` then the copy of the CRD will be skipped.

Reply:
I am fine with either option. For your reference, I used the similar format of CDO: https://github.com/openshift/cluster-dns-operator/blob/master/hack/update-generated-crd.sh